app/api/submission.py

```diff
@@ -1,104 +1,179 @@
 from hashlib import sha512
 import logging
 
+
 from fastapi import APIRouter
 from fastapi.responses import JSONResponse
 from requests import get
 
 from app.utils.complexity.squad_score import squad_score, scaler
 from app.utils.img_processing.google_api import GoogleAPI, NoTextFoundException
 from app.api.models import Submission, ImageSubmission
+from tensorflow import keras
+
+
 
 # global variables and services
 router = APIRouter()
 log = logging.getLogger(__name__)
 vision = GoogleAPI()
-
+model = keras.models.load_model('app/img_clustering_augmented_model.h5')
 
 @router.post("/submission/text")
 async def submission_text(sub: Submission):
-    """Takes a Submission Object and calls the Google Vision API to text annotate
-    the passed s3 link, then passes those concatenated transcriptions to the SquadScore
-    method, returns:
-
-    Arguments:
-    ---
-    `sub`: Submission - Submission object **see `help(Submission)` for more info**
-    Returns:
-    ---
-    ```
-    {"SubmissionID": int, "IsFlagged": boolean,"LowConfidence": boolean, "Complexity": int}
-    ```
-    """
-    transcriptions = ""
-    confidence_flags = []
-    # unpack links for files in submission object
-    for page_num in sub.Pages:
-        # re-init the sha algorithm every file that is processed
-        hash = sha512()
+    '''
+    This is a test endpoint that is a duplicate of the illustration submission endpoint 
+    that allows the web dev team to test the endpoints.
+    '''
         # fetch file from s3 bucket
```

Comment:
thank you for adding the inline documentation - it is the small attention to details like this that create transfer of knowledge in the moment and through time on a project